.gitignore

```diff
@@ -8,3 +8,4 @@ simpletest.js
 test2.js
 test3.js
 test4.js
+browser/bundle.js
```

Comment:
why is this file getting ignored? If it's only used during building, it should probably simply be deleted after building and copy/relocation instead of being in the gitignore.